modules/storage-persistent-storage-volume-encrypt-with-kms-key.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage-aws.adoc
+
```

Comment:
Can you set the content type to procedure here please?

Reply:
Addressed